### DISCUSSION TITLE
Presence detection when using manual temp so no preset

### DISCUSSION THREAD

**Discussion by thomase1234** [jmcollin78/versatile_thermostat#735] [General]
Hi all, 

Thanks for the great thermostat. It works really nice. Especially in combination with presence detection and the advanced scheduler.

I do have a situation, and I'd like to understand how other people solved it.

I'm using presence detection and it's working fine. F.e. if the Comfort temp is set, the thermostat switches to the _away temp as soon as no presence is detected. All good...

However: some people in the house like to change from the presets to a manual temp. After they leave and the house is empty, the thermostat keeps the house on the manually set temperature.

I thought about some solutions:
1. using an automation to switch to a preset as soon as the house is empty, but it's not trivial as the preset depends on the scheduler.
2. binding the manual thermostaat changes to the comfort setting so that any manual change basically changes the comfort temp. As a result, the preset comfort will always become activated if someone manually changes the temp.

How do you handle this scenario ? Any ideas ?

Thanks,
Thomas

**Comment by jmcollin78** [comment]
Hello @thomase1234 ,

Here is how I do:
1. I never need to switch to manual mode.

Thanks for your attention

**Comment by jmcollin78** [comment]
More seriously with all the automatism in VTherm and presets are made to avoid manual mode.

Instead of adding one degree, just set the preset to Boost. Instead of decreasing on degree, just set it to Eco.

If you never switch to manual mode, there is no issue. 

Note: that if your Scheduler periodically set the preset, the manual mode will be replaced by the preset and the VTherm goes back in normal behavior. This could be an idea: set the preset every 4 hours max, so that manual will never stay more than 4 hours.
